dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/Governors/Standard/Generic/GovCT2.mo

```diff
@@ -0,0 +1,352 @@
+  parameter Types.PerUnit Pm0Pu "Initial value of mechanical power";
+  parameter Types.PerUnit PRatePu = 0.017 "Ramp rate for frequency-dependent power limit" annotation( Dialog(tab = "Frequency dependent valve limit"));
+  parameter Types.PerUnit RClosePu = -99 "Minimum valve closing rate in pu/s" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.PerUnit RDownPu = -99 "Maximum rate of load limit decrease in pu" annotation( Dialog(tab = "Load limit controller"));
+  parameter Types.PerUnit ROpenPu = 99 "Maximum valve opening rate in pu/s" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.PerUnit RPu = 0.05 "Permanent droop in pu" annotation( Dialog(tab = "Main control path"));
+  parameter Integer RSelectInt "Feedback signal for droop" annotation( Dialog(tab = "Main control path"));
+  parameter Types.PerUnit RUpPu = 99 "Maximum rate of load limit increase in pu" annotation( Dialog(tab = "Load limit controller"));
+  parameter Types.Time tActuatorSeconds = 0.4 "Actuator time constant in s" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.Time tASeconds = 1 "Acceleration limiter time constant in s" annotation( Dialog(tab = "Acceleration limiter"));
+  parameter Types.Time tBSeconds = 0.1 "Turbine lag time constant in s" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.Time tCSeconds = 0 "Turbine lead time constant in s" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.Time tDGovSeconds = 1 "Governor derivative controller time constant in s" annotation( Dialog(tab = "Main control path"));
+  parameter Types.Time tDRatelimSeconds = 0.001 "Ramp rate limter derivative time constant in s";
+  parameter Types.Time tEngineSeconds = 0 "Transport time delay for diesel engine in s" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.Time tFLoadSeconds = 3 "Load limiter time constant in s" annotation( Dialog(tab = "Load limit controller"));
+  parameter Types.Time tLastValueSeconds = 1e-9 "Fast delay block time constant in s" annotation( Dialog(tab = "Main control path"));
+  parameter Types.Time tPElecSeconds = 2.5 "Electrical power transducer time constant in s" annotation( Dialog(tab = "Main control path"));
+  parameter Types.Time tSASeconds = 1e-6 "Temperature detection lead time constant in s" annotation( Dialog(tab = "Load limit controller"));
+  parameter Types.Time tSBSeconds = 50 "Temperature detection lag time constant in s" annotation( Dialog(tab = "Load limit controller"));
+  parameter Types.Time ValveMaxPu = 1 "Maximum valve position limit in pu" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.Time ValveMinPu = 0.175 "Minimum valve position limit in pu" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.Time WFnlPu = 0.187 "No load fuel flow in pu" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Boolean WFSpdBool = false "Switch for fuel source characteristic" annotation( Dialog(tab = "Turbine/engine")); 
```

Comment:
You gave a default value for all these parameters, it is good when using the model in OpenModelica, but in dynawo if we try to modify a default value in the .par, it might not be taken into account. This is something we are working on.
But I would suggest removing these default values and that you add in the description of every parameter the "typical value" similar to what we do in Dynawo.Electrical.Controls.WECC.Parameters.ParamsElectricalControl

Reply:
done